spec/libsass-todo-issues/issue_947/expected.compact.css

```diff
@@ -0,0 +1,2 @@
+@keyframes test { 10% { color: red; } }
+
```

Comment:
@xzyfer, would those extra line feeds at the end of file cause issue with test scripts? Normally, if it is CRLF then we add extra line feed so git squashes CRLF+CRLF+EOF (or LF+CRLF+EOF) to a single line feed (LF or CRLF) followed by EOF. If we don't have double CRLF, then git diff on GitHub shows a little red abnormal line termination marker (like [this](https://github.com/am11/node-sass/commit/ae70faeeb710f9a2a120f25de94e8c12d8eed414#diff-67dc932082c9767347dbd766ad547b5bR10)). But in case of double LF, git will retain double linefeeds. IMO, we should avoid this from happening.